<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPlainTextEdit{
border: 1px solid rgb(209, 209, 209);
background-color: white;
border-radius: 7;
padding:2;
max-height:25;
min-height:10;
max-width:200;
}
QComboBox{
border: 1px solid rgb(209, 209, 209);
background-color: white;
border-radius: 7;
padding:2;
max-height:25;
min-height:10;
max-width:200;
}
QLabel {
    font-size: 12px;
    font-family: &quot;Ms Shell Dlg 2&quot;;
    color: #000;
    font-weight: bold;
    text-decoration: underline;
}
QStackedWidget{
	border: 1px solid rgb(209, 209, 209);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_28">
        <item>
         <widget class="QPushButton" name="modelSettings_back_btn">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>25</width>
            <height>41</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: none;
	border: none;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/ICONS/462547553_520698087383761_137860076397263527_n.png</normaloff>:/ICONS/462547553_520698087383761_137860076397263527_n.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_18">
          <property name="font">
           <font>
            <family>Ms Shell Dlg 2</family>
            <pointsize>-1</pointsize>
            <weight>75</weight>
            <bold>true</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
    font-size: 20px;
    font-family: &quot;Ms Shell Dlg 2&quot;;
    color: #000;
    font-weight: bold;
    text-decoration: none;
}</string>
          </property>
          <property name="text">
           <string>Model Settings</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="verticalSpacing">
         <number>12</number>
        </property>
        <item row="2" column="0">
         <widget class="QLabel" name="label_32">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;MAX LEVEL 2 SHELTERS&lt;/span&gt; the maximum number of level 2 shelters that can be constructed or used&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="toolTipDuration">
           <number>10000</number>
          </property>
          <property name="text">
           <string>MaxLvl2Shelters</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_30">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;WEIGHT DISTANCE&lt;/span&gt; the shortest distance between the population and the shelters&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="toolTipDuration">
           <number>10000</number>
          </property>
          <property name="text">
           <string>WeightDistance</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QPlainTextEdit" name="textEdit_generations"/>
        </item>
        <item row="3" column="1">
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_20">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;GENERATIONS&lt;/span&gt; represents a stage in the genetic algorithm process. Higher generations indicate progress toward finding the best solution.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="toolTipDuration">
           <number>10000</number>
          </property>
          <property name="text">
           <string>Generations</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_19">
          <property name="font">
           <font>
            <family>Ms Shell Dlg 2</family>
            <pointsize>-1</pointsize>
            <weight>75</weight>
            <bold>true</bold>
            <underline>true</underline>
           </font>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;MODEL&lt;/span&gt; select to use which model is appropriate&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="toolTipDuration">
           <number>10000</number>
          </property>
          <property name="text">
           <string>Model</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="1">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="2">
         <widget class="QPlainTextEdit" name="textEdit_population"/>
        </item>
        <item row="5" column="2">
         <widget class="QPlainTextEdit" name="textEdit_wtCost"/>
        </item>
        <item row="8" column="2">
         <widget class="QPlainTextEdit" name="textEdit_mutation"/>
        </item>
        <item row="4" column="1">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="1">
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QComboBox" name="comboBox_modelType">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>16</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <item>
           <property name="text">
            <string>Bilevel No Transfer (BNT)</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_31">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;WEIGHT COST&lt;/span&gt; the cost of constructing or maintaining shelters&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="toolTipDuration">
           <number>10000</number>
          </property>
          <property name="text">
           <string>WeightCost</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPlainTextEdit" name="textEdit_wtDist"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_33">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;MAX SHELTERS&lt;/span&gt; the maximum number of shelters allowed&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="toolTipDuration">
           <number>10000</number>
          </property>
          <property name="text">
           <string>MaxShelters</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_27">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;POPULATION&lt;/span&gt; a set of potential solutions that evolves over multiple iterations/generations&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="toolTipDuration">
           <number>10000</number>
          </property>
          <property name="text">
           <string>Population</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_28">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;MUTATION RATE&lt;/span&gt; determines if a community is randomly selected to update shelter allocations based on a random value&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="toolTipDuration">
           <number>10000</number>
          </property>
          <property name="text">
           <string>Mutation Rate</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_29">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;AREA PER INDIVIDUAL&lt;/span&gt; represents the amount of space allocated for each individual in a shelter&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="toolTipDuration">
           <number>10000</number>
          </property>
          <property name="text">
           <string>AreaPerIndividual</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="2">
         <widget class="QPlainTextEdit" name="textEdit_maxShelters"/>
        </item>
        <item row="2" column="2">
         <widget class="QPlainTextEdit" name="textEdit_maxL2Shelters"/>
        </item>
        <item row="1" column="2">
         <widget class="QPlainTextEdit" name="textEdit_areaPerIndiv"/>
        </item>
       </layout>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="modelSettings_done_btn">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>161</width>
          <height>51</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #1C5739;
    color: white;
    padding: 5px;
    border: none;
    border-radius: 15px;
}

QPushButton:hover {
    background-color: #25754c;
}

QPushButton:pressed {
    background-color: #1C5739;
}
</string>
        </property>
        <property name="text">
         <string>DONE</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
